REFERENCE.md

```diff
@@ -272,6 +272,7 @@ interface JobOpts{
 interface RepeatOpts{
   cron?: string; // Cron string
   tz?: string, // Timezone
+  startDate?: Date | string | number; // Start data when the repeat job should start repeating (only with cron).
```

Comment:
while we are at it we could correct the typo "data" -> "date"

Reply:
Sure